### DISCUSSION TITLE
Sidebar

### DISCUSSION THREAD

**Discussion by Dr-Maspix** [filamentphp/filament#4004] [Help] [answered]
Dears ,
Can any one tell me how we can make multi menu inside each other 

Ex :

>Catalog
    >Products
        >Categories 

Regards

**Comment by devmatheus** [comment, marked as answer]
In your `ProductResource` and `CategoryResource`, you can use the `$navigationGroup` attribute, e.g.:

```php
class ProductResource extends Resource
{
    protected static ?string $navigationGroup = 'Catalog';
}

class CategoryResource extends Resource
{
    protected static ?string $navigationGroup = 'Catalog';
}
```

**Reply by Dr-Maspix** [reply]
Dear devmatheus ,
Thanks for your replay , 
but i mean i need to make mutli level , 

![image](https://user-images.githubusercontent.com/44633806/190156643-32bcb6a2-8371-4f56-8d4d-a5060a1e2ee8.png)

Like This one  ,

Regards
